### DISCUSSION TITLE
Exception for 3' rule for insertions/deletions around acceptor site

### DISCUSSION THREAD

**Discussion by sicotteh** [HGVSnomenclature/hgvs-nomenclature#161] [HGVS nomenclature proposals]
We propose to add an exception to the 3'rule to favor nomenclature that preserve the [c/t]AG acceptor recognition site. The 3' shifting rule is there to support the biology of translation. For cases of insertions in splicing, the 3' rule does not always agree with the biology. The biology of splicing is mediated with U[n] proteins for the A and B complex. The acceptor site consensus is [T]^n[C/T]AG. If a T is inserted in the T stretch, the 3' rule could shift the insertion into the beginning of the U2 binding domain.. but in reality the sequence around the [C/T]AG does not change and binding should not be affected. We propose that variants before the "AG" should be shifted 5' toward the center of the intron (up to center of intron) .. away from the core U2 binding site.

**Comment by jfjlaros** [comment]
The Mutalyzer team is currently working on something related; a way to decide which variants are propagated to the transcript, regardless of how they were normalised on the genome. This involves more than following a 3' or 5' rule, but essentially the idea is to move a variant out of the splice region into the intron or the exon. So while a variant may be described as intronic using the `c.` notation, it may still be present in the `r.` description. This approach works for both donor as well as acceptor sites and could in principle be used in a more general way to assess whether other features of the genome are affected.

I do not speak for the whole committee, but I think that this approach is more practical than adding exceptions to the nomenclature.

**Reply by jtdendunnen** [reply]
Every exception makes it more complex to apply HGVS nomenclature. Chances are therefore low that the nomenclature committee will add exceptions. Looking at your request, I do not see a real urge. I do not see why describing a variant as NM_xxxxxx.y:c.123-15del instead of NM_xxxxxx.y:c.123-4del would make a true difference. At best, the NM_xxxxxx.y:c.123-4del description warns the user to consider an effect on splicing, a valuable warning.

**Comment by NM-G08** [comment]
Hello,

I'm looking a variant that I think is similar to the situation described here.  I'm told that NC_000020.10(NM_001283009.2):c.1266+3_1266+80del is a consequence of the 3' rule being applied to NC_000020.10(NM_001283009.2):c.1228_1266+39del.

My first question is if the 3' rule applies to non-consecutive sequence.  I'm not convinced that c.1228_1266+39del is being shifted, particularly because I think these deletion is also described as NC_000020.10(NM_001283009.2):c.1266+29_1267-88del, but thought I would get the HGVS Committee's confirmation.

If c.1266+29_1267-88del is being shifted to c.1266+3_1266+80del per the HGVS 3' rule, then I think this issue/entry does apply to this example variant.  In this case, a deletion that overlaps with a canonical donor site suddenly looks like it's intronic.  The splicing algorithms have different predictions for all 3 versions of this deletion, which could be problematic here.

Thanks,
NM

**Reply by ifokkema** [reply]
I have checked with both VariantValidator and Mutalyzer, but these are three distinct deletions. They are not different descriptions of the same deletion.

> My first question is if the 3' rule applies to non-consecutive sequence.

No; the sequence should be consecutive.


> The splicing algorithms have different predictions for all 3 versions of this deletion, which could be problematic here.

As they are different deletions, this makes sense. However, although I am not educated on the internal workings of splicing prediction algorithms, _if_ different descriptions of one and the same variant would influence the outcome of a splicing prediction algorithm, it's evidence that the algorithm doesn't actually look at the resulting sequence of the deletion but instead takes the description's position into account. In my opinion, that should be avoided (for exactly this reason).

**Comment by NM-G08** [comment]
Thanks, I realized I had the wrong alteration.  It was actually c.1237_1266+48del, which I do think gets shifted to c.1266+3_1266+80del per the 3' rule.  However, thank you for the response regarding consecutive sequence.

Also appreciate your insight into the way the splicing algorithms work.  I'll have to look more closely at this example, but it doesn't seem it fits as part of the issue described in this thread anymore.

Thanks again!

**Reply by ifokkema** [reply]
Yes, indeed, `NC_000020.10(NM_001283009.2):c.1237_1266+48del` shifts to `NC_000020.10(NM_001283009.2):c.1266+3_1266+80del`. It's still a relevant example, I think, as also here, one may assume the two different descriptions may suggest different outcomes on the splicing of this transcript. If a splicing algorithm provides different predictions for these two descriptions, it highlights a shortcoming of this algorithm.
